python/ray/tune/suggest/hyperopt.py

```diff
@@ -6,8 +6,8 @@
 import copy
 import logging
 try:
-    hyperopt_logger = logging.getLogger("hyperopt")
-    hyperopt_logger.setLevel(logging.WARNING)
+    logger = logging.getLogger(__name__)
```

Comment:
oh, this is supposed to be specifically only target the hyperopt logger. Can you revert?

I would add a separate `logger = logging.getLogger(__name__)` on like line 17. 

So to be clear, the hyperopt logger is only instantiated to get rid of HyperOpt warnings; we should use the named logger for the rest of the file.